src/utilities.jsx

```diff
@@ -0,0 +1,13 @@
+import {IconWrapper} from './Wrapper'
+import React from 'react'
+import ReactDOMServer from 'react-dom/server'
+
+export const getSvgIcon = (svgIcon, iconProps) => {
+  const iconElement = React.createElement(svgIcon, {
+    ...iconProps
+  })
+  
+  const iconWrapper = React.createElement(IconWrapper, {}, iconElement)
+  
+  return ReactDOMServer.renderToStaticMarkup(iconWrapper)
```

Comment:
We do something similar in safe-app (Footer.jsx). But instead we use` __dangerouslySetInnerHtml`